nbs/docs/tutorials/20_conformal_prediction.ipynb

```diff
@@ -0,0 +1,230 @@
+    "(\n",
+    "  spark_df\n",
+    "  .repartition(id_col)\n",
+    "  .sortWithinPartitions(id_col, time_col)\n",
+    "  .write\n",
+    "  .partitionBy(id_col)\n",
+    "  .parquet(out_dir)\n",
+    ")\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The DataLoader class still expects the static data to be passed in as a single DataFrame with one row per timeseries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "static = AirPassengersStatic.rename(columns={'unique_id': 'id_col'})\n",
+    "static"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model training\n",
+    "\n",
+    "We now train a NHITS model on the above dataset. \n",
+    "It is worth noting that NeuralForecast currently does not support scaling when using this DataLoader. If you want to scale the timeseries this should be done before passing it in to the `fit` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "horizon = 12\n",
+    "stacks = 3\n",
+    "models = [NHITS(input_size=5 * horizon,\n",
+    "                h=horizon,\n",
+    "                futr_exog_list=['trend', 'y_[lag12]'],\n",
+    "                stat_exog_list=['airline1', 'airline2'],\n",
+    "                max_steps=100,\n",
+    "                stack_types = stacks*['identity'],\n",
+    "                n_blocks = stacks*[1],\n",
+    "                mlp_units = [[256,256] for _ in range(stacks)],\n",
+    "                n_pool_kernel_size = stacks*[1])]\n",
+    "nf = NeuralForecast(models=models, freq='ME')\n",
+    "nf.fit(df=files_list, static_df=static, id_col='id_col')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Forecasting\n",
+    "\n",
+    "When working with large datasets, we need to provide a single DataFrame containing the input timesteps of all the timeseries for which wish to generate predictions. If we have future exogenous features, we should also include the future values of these features in the separate `futr_df` DataFrame. \n",
+    "\n",
+    "For the below prediction we are assuming we only want to predict the next 12 timesteps for Airline2."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "valid_df = valid[valid['id_col'] == 'Airline2']\n",
+    "# we set input_size=60 and horizon=12 when fitting the model\n",
+    "pred_df = valid_df[:60]\n",
+    "futr_df = valid_df[60:72]\n",
+    "futr_df = futr_df.drop([\"y\"], axis=1)\n",
+    "\n",
+    "predictions = nf.predict(df=pred_df, futr_df=futr_df, static_df=static)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| hide\n",
```

Comment:
I think this is still unfinished, right? 

I think the example would benefit from a plot showing the conformal intervals, ideally compared against a second setup where we just do DIstributionLoss("Normal") or something like that, displaying the width of the uncertainty intervals side-by-side (you can copy the code for such interval plots from a model page, e.g. BiTCN has it